sinks/api/http/endpoint.go

```diff
@@ -50,14 +52,19 @@ func addEndpoint(svc sinks.SinkService) endpoint.Endpoint {
 			return nil, err
 		}
 
+		// Eu posso tentar setar uma flag para sinalizar qual o tipo do método que está
```

Comment:
Acho que a gente pode apagar isso aqui, ou mudar pro ingles, se for importante para voce manter o comentario :)

Reply:
Boa amanda, isso é garbage. vou remover!